ledger/sandbox/src/main/scala/com/digitalasset/platform/store/ActiveLedgerStateManager.scala

```diff
@@ -150,10 +143,7 @@ class ActiveLedgerStateManager[ALS <: ActiveLedgerState[ALS]](initialState: => A
                   witnesses = disclosure(nodeId),
                   // we need to `getOrElse` here because the `Nid` might include absolute
                   // contract ids, and those are never present in the local disclosure.
-                  divulgences = (localDivulgence
-                    .getOrElse(nodeId, Set.empty) diff nc.stakeholders).toList
-                    .map(p => p -> transactionId)
-                    .toMap,
+                  divulgences = Map.empty,
```

Comment:
Would this affect privacy-aware participants?

Reply:
No, because this field was filled with data coming from `localDivulgence`, which is proven to be empty at this stage (see https://github.com/digital-asset/daml/pull/4424/files#diff-63d033a4194960839071d50782fc9920L291)

Reply:
But the comment above it should be reworded...